jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -171,6 +171,12 @@ private static void completeInitialization() {
 			Runtime.getRuntime().addShutdownHook(new Thread(runnable, "CommonCleanerShutdown", true, false, false, null)); //$NON-NLS-1$
 		}
 		/*[ENDIF] JAVA_SPEC_VERSION >= 9 */
+/*[IF CRAC_SUPPORT]*/
+		if (openj9.internal.criu.InternalCRIUSupport.isCRaCSupportEnabled()) {
```

Comment:
is this after java.base is created?

Reply:
With `-Xtrace:print={module.15} -Xtrace:print=mt,methods={openj9/internal/criu/InternalCRIUSupport.isCRaCSupportEnabled}`, a testcase `org.openj9.criu.TestJDKCRAC` run has following output:
```
20:14:10.705*0x1aa00  module(j9scar).15       - JVM_DefineModule() defines module java.base (0x00007F6AF009A1B8)
20:14:10.752 0x1aa00  module(j9scar).15       - JVM_DefineModule() defines module java.instrument (0x00007F6AF017F478)
.... many other module defines
20:14:10.767 0x1aa00  module(j9scar).15       - JVM_DefineModule() defines module java.xml.crypto (0x00007F6AF01EEC08)
20:14:11.136 0x1aa00              mt.3        > openj9/internal/criu/InternalCRIUSupport.isCRaCSupportEnabled()Z bytecode static method
20:14:11.136 0x1aa00              mt.9        < openj9/internal/criu/InternalCRIUSupport.isCRaCSupportEnabled()Z bytecode static method
```
So `java.base` has been created before `InternalCRIUSupport.isCRaCSupportEnabled()` invocation.
BTW, `TestJDKCRAC` passed as well.